addons/website_event/static/src/scss/event_templates_page.scss

```diff
@@ -114,3 +123,10 @@
 .o_wevent_hide_sponsors .o_wevent_sponsor_wrapper {
     display: none !important;
 }
+
+.o_event_add_to_calendar_btns a {
```

Comment:
Would be great to keep it in a namespace, we try to avoid adding classes that are not under a parent class to avoid conflicts and keep rules organized and specific.

Reply:
I put it outside the parent because we want the same css for the website_event_sale confirmation cards :/
Do you think I should duplicate it in the website_event_sale module so that we keep it inside its parents ?

Reply:
MMmh, ok. Would be worth a small comment then maybe ? Like ``// Used in wevent and wevent_sale`